app/src/main/java/com/example/ex3/Quotes.java

```diff
@@ -0,0 +1,21 @@
+package com.example.ex3;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Quotes {
+    private final String[] quotes = {
+            "Vini Vidi Vici",
+            "Carpe Diem",
+            "YOLO",
+            "Que Sera Sera",
+            "Ce la vie",
+            "Asta la vista baby",
+            "Work Hard Play Hard"
+    };
+
+    String getRandomQuote(){
```

Comment:
you could have made it a static function, it will be better coding in that specific scenario